webapp/components/search_bar.jsx

```diff
@@ -171,7 +171,16 @@ export default class SearchBar extends React.Component {
 
     handleSubmit(e) {
         e.preventDefault();
-        this.handleSearch(this.state.searchTerm.trim());
+        var terms = this.state.searchTerm.trim();
```

Comment:
@jwilander fixed in https://github.com/mattermost/platform/pull/7014/commits/3f4101733cb992479046769c70f33c90ae7a78a8